### ISSUE TITLE
About dataset splits

### ISSUE THREAD

**Issue by tarekegn82** [Milkomeda98/CKD-TransBTS#4]
Dear Jianwei, 
Thank you for your contribution to medical image segmentation research with this paper. I am trying to reproduce your model and I have a question regarding the train, validation, and test lists of BraTS21. Do I need to do the split manually or are there any programmable options I could use to automate the process? I have Training Data from the "RSNA_ASNR_MICCAI_BraTS2021_TrainingData_16July2021". My training experiment on the whole dataset works fine, but I would like to run the test as well.

**Comment by sword98** [comment]
Thank you for your attention. In this work, we have split the 'RSNA_ASNR_MICCAI_BraTS2021_TrainingData' dataset into train set, validation set and test set, and have released corresponding sample name lists in `dataset/` folder. You can write a program to split the dataset according to the provided sample name list.  Alternatively, you can import the ` scikit-learn` package and use K-fold cross-validation to verify the performance of the model.
In addition, the BraTS2021 competition is still continuing, you can download the 'RSNA_ASNR_MICCAI_BraTS2021_ValidationData' from the [competition platform](https://www.synapse.org/#!Synapse:syn25829067/wiki/610864) and submit the segmentation results to the platform to validate the performance of your model.

**Comment by tarekegn82** [comment]
Thank you for your helpful response. I have gained access to the `Validation` dataset however, I have some errors due to missing Modalities under the validation dataset. To solve this issue, I'm going to manually split the `trainingData` into three sets as you did.

I need your suggestion on the software you used for some of your works:
1. Which Software is good for visualizing 3D MRI modalities
2. Which Software is good for Designing Model Architecture `(e.g. CKD-TransBTS.png)`

Thank you

**Comment by sword98** [comment]
1、I personally recommend ITK-SNAP, you can download and install it [here](http://www.itksnap.org/pmwiki/pmwiki.php)
2、I usually use PowerPoint to draw, maybe you can refer to the style of some excellent papers and find some good ideas from them.

**Comment by tarekegn82** [comment]
Thank you

**Comment by faizan1234567** [comment]
@tarekegn82 how did you split the dataset into train, validation and test sets? any idea how to split the dataset? or you used the k fold cross validation ? What does it mean by the k fold cross validation ? there is no such thing mentioned in the paper?

**Comment by tarekegn82** [comment]
Hi @faizan1234567 , In this folder named [dataset](https://github.com/Milkomeda98/CKD-TransBTS/tree/main/dataset) you can find dataset splits for train, val and test. when you put the BraTS2021 dataset in the correct directory and edit the training script based on your own directory, the training will work just perfectly.
